content/components/action-list.mdx

```diff
@@ -0,0 +1,157 @@
+- **Action list divider:** …
+- **Action list section header:** …
+
+<img alt="Action list item structure" src="https://user-images.githubusercontent.com/223966/118818435-3bacd400-b8ac-11eb-8691-ab9cbba73249.png" />
+<Caption>Action list item structure</Caption>
+
+
+
+## Application
+
+### In overlays
+
+### In pages
+
+
+-->
+
+## Anatomy 
+
+<img src="https://user-images.githubusercontent.com/293280/122128726-d7692b80-cde9-11eb-8e5b-2f4e2eb4b4a1.png" alt="Action list anatomy" />
+
+<img src="https://user-images.githubusercontent.com/293280/121974972-7718b200-cd35-11eb-8c04-1fffbc5deedc.png" alt="Action list item anatomy" />
+
+
+## Options
+
+<Grid gridTemplateColumns={['1fr', null, null, null, '1fr 1fr']} gridGap={5}>
+    <div>
+        <Flex justifyContent="center">
+            <img width="464" alt="Action list sizes" src="https://user-images.githubusercontent.com/293280/121766871-c1b5e680-cb09-11eb-8835-bfec4c80e45a.png" />
+        </Flex>
+    </div>
+    <div>
+        <Heading fontSize={3}>Sizes</Heading>
+        <Flex as="p">Action list items support three different sizes: small, medium, and large. The small size is the default and most common option. Medium sizes work well for relaxed local navigation, while large sizes can support items that need more breathing room.</Flex>
+        <Flex as="p">Sizes only grow vertically. This behavior keeps the content aligned among items, and retains horizontal space for density.</Flex>
+        <Flex as="p">On touch devices, the large size is used at all times to ensure usability when tapping.</Flex>
+        <Flex as="p">Avoid mixing different sizes in the same list.</Flex>
+    </div>
+    <div>
+        <Flex flexDirection="column" justifyContent="center">
+            <Box>
+                <img width="464" alt=" " src="https://user-images.githubusercontent.com/293280/121966246-226c3b80-cd23-11eb-856b-32bde42cb60f.png" />
+                <Caption>Use leading visuals to represent system sections, features, or options.</Caption>
+            </Box>
+            <Box>
+                <img width="464" alt=" " src="https://user-images.githubusercontent.com/293280/121966249-2304d200-cd23-11eb-9bbf-fbc1ef2773b1.png" />
+                <Caption>Use leading visuals in important menu items.</Caption>
+            </Box>
+            <Box>
+                <img width="464" alt=" " src="https://user-images.githubusercontent.com/293280/121966449-8c84e080-cd23-11eb-930f-9397ae2ea433.png" />
+                <Caption>Use leading visuals to represent content types.</Caption>
+            </Box>
+        </Flex>
+    </div>
+    <div>
+        <Heading fontSize={3}>Leading visual</Heading>
+        <Flex as="p">Leading visuals are optional and appear at the start of an item. They can be octicons, avatars, and other custom visuals that fit a small area.</Flex>
```

Comment:
Would it look odd if the leading visuals section was text on the left and images on the right?